### ISSUE TITLE
Geometry Nodes "Shift-A" Crash

### ISSUE THREAD

**Issue by animatedai** [bradklingensmith/project-glue#1]
Encountered this error while pressing "Shift-A" in Geometry Nodes.

```
Error:
  Traceback (most recent call last):

  File "C:\Users\Brad\Documents\blender\project-manager\scripts\addons\Project GLUE\error_handling.py", line 117, in wrapped_fn
    return fn(a1, a2, *args, **kwargs)

  File "C:\Users\Brad\Documents\blender\project-manager\scripts\addons\Project GLUE\ui\context_menus.py", line 46, in _add_to_NODE_MT_add
    'Node Tree' if tree_type is None else mybpy.get_native_rna_name_for_id_type(tree_type))

  AttributeError: module 'Project GLUE.mybpy' has no attribute 'get_native_rna_name_for_id_type'


Environment:
  Blender Version: (3, 4, 0)
  Project Glue Version: (1, 0, 1)
  Operating System: Windows-10-10.0.19045-SP0
  
  Installed Addons:
    io_anim_bvh
    io_curve_svg
    io_mesh_ply
    io_mesh_stl
    io_mesh_uv_layout
    io_scene_fbx
    io_scene_gltf2
    io_scene_obj
    io_scene_x3d
    cycles
    pose_library
    node_wrangler
    space_view3d_copy_attributes
    io_import_images_as_planes
    AI-Render
    development_icon_get
    Shortcutable Reload Scripts
    Project GLUE
    NodePreview
    blender_visual_scripting_addon
  
  Blend Files in Glue Project:
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\1x1x1 Ball - Standalone.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\1x1x1 Ball.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\1x1x1 Cube.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\3x3x8 Selector.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Add Cube Unit Material Attributes.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Arrow.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Audio Wave.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Backdrop.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Box Meter.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Capture Tensor Attributes.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Checkmark.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Connect With Arrow Path.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Conv2D Layer.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Create Filter Window.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Create Text.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Cube Tensor Unit Glass.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Cube Tensor Unit Metal.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Cube Tensor Unit Rubber.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Easing.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Exploding Tensor.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Eye Pair.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Fade-in Grid.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Filters.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Frame 2D.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Functions - Shader.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Functions.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Geo Nodes PB HSL.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Get Calc Activation.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Ground.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Instanced Tensor.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Large Tensor.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Layer text backup.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Layer.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Lerp.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Mirror.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\PB HSL.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Pop-in Grid.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Pop-in Rotation.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Pop-in.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\primitives.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Range Line.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Set Cube Unit Materials.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Shader - Get Tensor Color.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Shell Text.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Shell.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\speaker.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\string.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Surround Many.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Surround.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Tensor Footprint.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Tensor Placeholder.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Tensor World.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Tensor.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Text Range Line.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Textured Tensor.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Theme.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\To Sphere.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Transformer Key.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Transformer Sinusoid Encoding.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Wireframe Cube.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\primitives\Word Indexer.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Filter Count.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Groups.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Introduction.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Kernel Size.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Padding.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Convolution - Theory - Stride.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\demo.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\GIFs.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\images-as-tensors.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\layers.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Logo.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\NN - Introduction.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Outro.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\Scene Template.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\render\Flat Rainbow.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\transformers\High Level Task.blend
    C:\Users\Brad\Documents\blender\animated-ai\v2\scenes\transformers\Transformer Block.blend
```

**Comment by animatedai** [comment]
[Jira Ticket](https://bradklingensmith.atlassian.net/browse/BPG-9)

**Comment by animatedai** [comment]
Fixed in v1.02
